java/src/Ice/src/main/java/com/zeroc/Ice/Acceptor.java

```diff
@@ -2,8 +2,11 @@
 // Copyright (c) ZeroC, Inc. All rights reserved.
 //
 
-package com.zeroc.IceInternal;
+package com.zeroc.Ice;
 
+/**
+ * @hidden Kept public because it's used by IceBT and SSL.
```

Comment:
I find the "Kept public" doc-comment odd. It's really about this PR. Going forward, it's simply "Public because ...". Same for all the other "Kept public" comments.

Reply:
Good point, I removed the `Kept` part of the comment.